src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -40,6 +53,13 @@ private static Rotation getRotation(Block block) {
         return rotation;
     }
 
+    /**
+    * Give the new side of the block based on its' rotation
```

Comment:
Slightly confusing as well. You say give the new side, but the method returns more than one side?